addons/resource.language.en_gb/resources/strings.po

```diff
@@ -20716,7 +20720,19 @@ msgctxt "#36631"
 msgid "Force weak SMBv1 security for compatibility with the USB sharing features on some WiFi routers and NAS devices."
 msgstr ""
 
-#empty strings from id 36632 to 36898
+#. Text in confirm dialog when enabling setting "Services -> Control -> Allow remote control via HTTP"
+#: xbmc/network/NetworkServices.cpp
+msgctxt "#36632"
+msgid "Anyone who has access to the web interface will be able to completely control this application and, therefore, this device. You should protect the interface with a strong password that must be set below. Proceed?"
```

Comment:
I don't worry that much about a proper SSL-secured server with a good password, but maybe you are right security is not really our strong suit and there might be plenty bugs in, for instance, our HTTP handling. So should I add that point to the string? It will probably become too long to fit on the screen then, though.

Reply:
Ya I rarely trust anything exposed to the internet but the heavily used, battle-tested software web servers.

Haha, I think if you removed "that must be set below" there would be just enough room...

It may work to add it to the description instead of the confirmation. The description says "web server" which really does sound like it relates to the internet.